components/bl-multiple-select-component/README.md

```diff
@@ -14,9 +14,10 @@ Multiple Select is a component of Backendless UI-Builder designer. This componen
 | Placeholder         | *Text*                                                       | 'Placeholder'        |                           | NO           | YES        | This handler allows you to specify a placeholder for a component.                                                                                                        |
 | Select All Checkbox | *Checkbox*                                                   | `true`               | Select All Checkbox Logic | YES          | YES        | This handler allows you to add a checkbox to select all options.                                                                                                         |
 | Select All Label    | *Text*                                                       | 'Select all options' | Select All Label Logic    | YES          | YES        | This handler allows you to specify label for "Select All Checkbox".                                                                                                      |
-| Variant             | *Select* <br/>'outlined' \| <br/>'filled' \| <br/>'standart' | 'outlined'           |                           | NO           | YES        | This handler allows you to select a variant of the component.                                                                                                            |
-| Type                | *Select* <br/>'default' \| <br/>'checkmark' \| <br/>'chip'   | 'checkmark'          |                           | NO           | YES        | This handler allows you to select a type of the component.                                                                                                               |
-| Options             | *JSON*                                                       | `[]`                 | Options Logic             | YES          | YES        | This handler allows you to add options to the component. Watch [Codeless Examples](#Examples). Signature of options: list of objects { label: `String`, value: `String`} |
+| Variant             | *Select* <br/>'Outlined' <br/>\| 'Filled' <br/>\| 'Standart' | 'Outlined'           |                           | NO           | YES        | This handler allows you to select a variant of the component.                                                                                                            |
```

Comment:
As I see, values used in  component.json is lowercased.

https://github.com/Backendless/ui-builder-library/blob/main/components/bl-multiple-select-component/component.json